num/src/bigint.rs

```diff
@@ -696,7 +859,7 @@ macro_rules! construct_bigint {
             type Output = $name;
 
             fn div(self, other: T) -> $name {
-                self.div_rem(other.into()).0
+                self.div_rem(other.into()).unwrap().0
```

Comment:
We never do `unwrap` in non-test code and should use `expect()`. Here, something like
```suggestion
                self.div_rem(other.into()).expect("overflow during big integer division").0
```
will work better